### DISCUSSION TITLE
Session Storage?

### DISCUSSION THREAD

**Discussion by Laf303** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#683] [Q&A]
**Hello**
### 

> I was looking at the project and the whole project when I found this error,

_I hope and help me_ **＞︿＜** 

![image](https://user-images.githubusercontent.com/102272461/159833181-3123897b-ffb4-4fe6-a49f-5b50b7de9dc1.png)

**Comment by im-coder-lg** [comment]
Could be this(most probable of my answer list in my mind regarding this issue, tops off at 91% in probability of a correct answer): https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/discussions/682#discussioncomment-2413462

Don't thank me, thank the community for finding the fix, I just extended it over and yonder(who says that these days? Maybe only me)
